astro-client-core/client.go

```diff
@@ -41,6 +41,7 @@ func requestEditor(ctx httpContext.Context, req *http.Request) error {
 	req.Header.Add("x-astro-client-identifier", "cli")
 	req.Header.Add("x-astro-client-version", version.CurrVersion)
 	req.Header.Add("x-client-os-identifier", os+"-"+arch)
+	req.Header.Add("User-Agent", fmt.Sprintf("astro-cli/%s", version.CurrVersion))
```

Comment:
isn't this the same as 

req.Header.Add("x-astro-client-identifier", "cli")
req.Header.Add("x-astro-client-version", version.CurrVersion)

We use this for core-api to populate audit logs for cli requests

Reply:
yeah it's the same use case. just felt like we should use User-Agent since it's more standard. Core API will start to get requests from
more places when it becomes public so we should start tracking/logging User-Agent there. this will help with that

Reply:
we already log the user-agent in our logging library btw